src/imdb_data.py

```diff
@@ -0,0 +1,89 @@
+'''
+    File name: imdb_data.py
+    Author: Henry Letton
+    Date created: 2021-03-09
+    Python Version: 3.8.3
+    Desciption: Getting data from imdb source
+    NOTE: Could add date filter to only add recent films and cutdown run time
+'''
+
+import requests
+import gzip
+import shutil
+import pandas as pd
+from src.db_fns import create_engine2, df_to_sql_db, sql_db_to_df
+import numpy as np
+import hashlib
+
+# Move to correct folder
+import os
+os.getcwd()
+os.chdir('C:/Users/henry/OneDrive/Documents/AXA Insurance/Henry and Max Projects/What-to-watch')
```

Comment:
Probably don't want hand coded local paths

Reply:
True, but the code is only a vague draft and does not work 100%.
I saving it in case we ever need to use imdb data.
I can still remove this part on principle.